hax/hax/handler.py

```diff
@@ -217,6 +217,21 @@ def update_process_failure(self, planner: WorkPlanner,
                                                   chp_type=proc_type,
                                                   chp_pid=0,
                                                   fid=state.fid))
+                                LOG.debug('----xxxx Failure of local '
+                                          'node-process=%s! %s',
+                                          state.fid, proc_status)
+
+                                if current_status == ObjHealth.OFFLINE:
+                                    LOG.debug('----xxxxxx inside stopped 3-0')
+                                    cns = self.consul
+                                    node = cns.get_process_node(state.fid)
+                                    node_fid = cns.get_node_fid(node)
+                                    LOG.debug('----xxxxxx inside '
```

Comment:
Not able to understand what this log says.

Reply:
Sorry, these logs are for debugging purposes while testing various scenarios. I'll remove these in final patch.

Reply:
removed them